### PR TITLE
feature:version source and target on helm_promote

### PR DESCRIPTION
**Pull request by paulopontesm**

### CODE REVIEW

File: src/jobs/helm_promote.yaml [hunk 1]
```diff
@@ -1,8 +1,13 @@
 description: "Set version and publish to chart museum"
 
 parameters:
-  version:
-    description: "Name of the helm chart"
+  source_version:
+    description: "Name of the source helm chart"
```
Comment (rtfpessoa) [inline src/jobs/helm_promote.yaml]: This is very confusing. What do you mean by source version? What does version have to do with name of the chart?

File: src/jobs/helm_promote.yaml [hunk 2]
```diff
@@ -1,8 +1,13 @@
 description: "Set version and publish to chart museum"
 
 parameters:
-  version:
-    description: "Name of the helm chart"
+  source_version:
```
Comment (rtfpessoa) [inline src/jobs/helm_promote.yaml]: Why do we need to versions?

Comment (paulopontesm) [reply src/jobs/helm_promote.yaml]: I want to promote version `0.6.0-RC-1` in `incubator` to `0.6.0` in `stable`

File: src/jobs/helm_promote.yaml (hunk 1 above)
Comment (paulopontesm) [reply src/jobs/helm_promote.yaml]: Its a helm_promote step. We have a source and a target.
Downloads the source and uploads to the target

Comment (paulopontesm) [reply src/jobs/helm_promote.yaml]: Do you have a suggestion of better names?

File: src/jobs/helm_promote.yaml (hunk 2 above)
Comment (rtfpessoa) [reply src/jobs/helm_promote.yaml]: Let me see if I understand.
You have a version and you wont republish a new one.
You will kind of import he version from one repository of charts to the other?

If this is it I would let this very clear in the `description` of `helm_promote`. Something in the lines of `Promote version from one chart repository to another (e.g.: incubator -> stable)`

Regarding the `source_version` and `target_version` we can say something along the lines of `Name of the version in the source charts repository`

File: src/jobs/helm_promote.yaml (hunk 1 above)
Comment (rtfpessoa) [reply src/jobs/helm_promote.yaml]: I wrote more in the other comment. Please check.
After the extra context my main feedback is that you created a name `promote` that does not exist in helm and when you do that you should document it really well for people to use it consciously and know what are the effects bellow.

**Pull request merged**